sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/export/CollectionHandle.java

```diff
@@ -37,15 +38,13 @@ private CollectionHandle(int index) {
   }
 
   /**
-   * Construct a new (mutable) set consistenting of the passed in collection handles.
+   * Construct a new (mutable) set consisting of the passed in collection handles.
    *
    * <p>Used by tests.
    */
   static Set<CollectionHandle> of(CollectionHandle... handles) {
     Set<CollectionHandle> result = mutableSet();
-    for (CollectionHandle handle : handles) {
-      result.add(handle);
-    }
+    result.addAll(Arrays.asList(handles));
```

Comment:
Can you revert this one? Doesn't seem to be an obvious improvement

Reply:
@anuraaga reverted.